dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilder.java

```diff
@@ -57,13 +60,17 @@
   private static final CharMatcher LOWER_ALPHA_NUMERIC =
       CharMatcher.inRange('a', 'z').or(CharMatcher.inRange('0', '9'));
 
+  private final List<UnresolvableArtifactProblem> artifactProblems = new ArrayList<>();
+
+  private final ImmutableList<RemoteRepository> repositories;
```

Comment:
Repository URLs are the configuration of DependencyGraphBuilder.

Reply:
This one does make sense as a field.